devel/tldr-pages/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tldr-pages tldr 5a38d5f9f7dff0111fa756fdbbfc1546aa6905c7
+github.tarball_from archive
+
+name                tldr-pages
+categories          devel
+license             MIT
+platforms           any
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+description         The tldr-pages project is a collection of \
+    community-maintained help pages for command-line tools
+long_description    {*}${description}. This package installs an archive \
+    of the latest help pages to your system.
+
+depends_build-append port:npm11
+
+checksums           rmd160  ec0672e56cec69ce79ceb1b6f2ac933db7585b5c \
+                    sha256  1c84ec26be35301124931077545c34063bb83ec7832409790b8c3631f6ee267e \
+                    size    5221478
+
+use_configure no
+build {
+    system -W ${worksrcpath} "npm ci"
+    system -W ${worksrcpath} "npm run build-index"
+    system -W ${worksrcpath} "zip -q -r tldr.zip pages* LICENSE.md index.json"
+}
```

Comment:
can/should this be using the `npm` PortGroup instead (disclaimer: I have no experience with `npm` build/packages/PortGroup)?